packages/blade/src/components/Input/OTPInput/__tests__/__snapshots__/OTPInput.web.test.tsx.snap

```diff
@@ -332,7 +332,7 @@ exports[`<OTPInput /> should render 1`] = `
                   inputmode="decimal"
                   placeholder=""
                   required=""
-                  type="text"
+                  type="numeric"
```

Comment:
Numeric?

Reply:
Keyboard type is decimal, wanted to make the type similar too. Should've done thing before only

Reply:
Related to this right? https://github.com/razorpay/blade/issues/786 
Hopefully no one is depending on OTP being alphanumeric.

Reply:
Probably not. We can revisit this if we see someone facing any issues. Don't think we need to work on this right now

Reply:
So `numeric` won't really stop that. We might have to add `pattern="\d*"` for this.
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/30949385/204991463-fd7f93cd-4739-416b-8a80-a110375f087e.png">

Reply:
Yeah, it won't stop it. Just made it consistent with the keyboardType. Whether we want to stop it or not can be tackled separately. Lets scope this PR to just addition of masked input. I added `numeric` to make it consistent with they keyboardType but I can revert it if its an issue 😅

Reply:
Oh yes yes makes sense. Only this concern then https://github.com/razorpay/blade/pull/897#discussion_r1036766905. Rest looks good to me.

Reply:
AHH makes sense. So numeric will change the keyboard behaviour not the input's validation.